distributed/dashboard/components/scheduler.py

```diff
@@ -346,9 +345,10 @@ def update(self):
             # FIXME https://github.com/dask/distributed/issues/4675
             #       This causes flickering after adding workers and when enough memory
             #       is spilled out
```

Comment:
If these changes close https://github.com/dask/distributed/issues/4675, then we can remove this comment